docs/cli/reference.md

```
@@ -655,6 +661,65 @@ Move session to group
 
 
 
+## `aoe plugin`
+
+Manage plugins (list, info, enable, disable)
+
+**Usage:** `aoe plugin <COMMAND>`
+
+###### **Subcommands:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)